### ISSUE TITLE
Blog: How AFOLU sector responsible for GFG emission and removal

### ISSUE THREAD

**Issue by Sukriti-sood** [moja-global/community-website#251]
### Is your feature request related to a problem? Please describe.

Most of the contributors may want to understand FLINT and moja-global but got confused due to less understanding about sources  of GFG.

### Describe the solution you'd like.

I want to write a blog explaining how AFOLU sector is also major source of gfg emission.

**Comment by chicken-biryani** [comment]
@Sukriti-sood this sounds like a great idea.

Another good blog might be-

To explain the basic terms that one may encounter. Such as pools, fluxes etc.

Or maybe that can be included in this blog itself. What do you think?

**Comment by Sukriti-sood** [comment]
@chicken-biryani I think separate blog will be good.

**Comment by chicken-biryani** [comment]
@Sukriti-sood 

Awesome. Would you be okay with creating a seperate issue for the same?

**Comment by padmajabhol** [comment]
> @Sukriti-sood
> 
> Awesome. Would you be okay with creating a seperate issue for the same?

Can I work on the other blog? The one that could explain about pool, fluxes, etc?

**Comment by chicken-biryani** [comment]
@padmajabhol definitely!

Just make an issue before hand. Ask someone to assign it to you. (You know the drill 😉).

I think this blog would be extremely helpful.

What you could probably do is... Go through Flint-wiki and the document Mohit has worked on. (I'll link it when I find it. Please, remind me if I forget. 😅)

You'll see a lot of terms such as GHG, Pools, Fluxes-- that probaby don't make sense to people from the technical background even after reading the Flint-wiki (Gets intimidating).

So you could maybe break it down for people like you and me? The Eli5 (Explain like I am 5) way.

It doesn't have to be all at once. Just pick up a few terms and try to tackle them.

(And later there could be follow up blogs with more terms.)

Cc: @shubhamkarande13 @aornugent @HarshCasper

**Comment by HarshCasper** [comment]
> @Sukriti-sood this sounds like a great idea.
> 
> Another good blog might be-
> 
> To explain the basic terms that one may encounter. Such as pools, fluxes etc.
> 
> Or maybe that can be included in this blog itself. What do you think?

We already have them here: https://docs.moja.global/en/latest/Understanding-FLINT/moja-flint/index.html

Why do we need a blog?

**Comment by HarshCasper** [comment]
Hi @Sukriti-sood 

Here's a cool paper by the Werner Kurz, chair of the Technical Steering Committee, shared by @aornugent on the #outreachy channel of Slack:

[annurev-environ-012320-054809.pdf](https://github.com/moja-global/community-website/files/7728817/annurev-environ-012320-054809.pdf)

**Comment by chicken-biryani** [comment]
> > @Sukriti-sood this sounds like a great idea.
> > 
> > Another good blog might be-
> > 
> > To explain the basic terms that one may encounter. Such as pools, fluxes etc.
> > 
> > Or maybe that can be included in this blog itself. What do you think?
> 
> We already have them here: https://docs.moja.global/en/latest/Understanding-FLINT/moja-flint/index.html
> 
> Why do we need a blog?

I feel that the content can get a little intimidating.

I may be wrong.

**Comment by padmajabhol** [comment]
> @padmajabhol definitely!
> 
> Just make an issue before hand. Ask someone to assign it to you. (You know the drill 😉).
> 
> I think this blog would be extremely helpful.
> 
> What you could probably do is... Go through Flint-wiki and the document Mohit has worked on. (I'll link it when I find it. Please, remind me if I forget. 😅)
> 
> You'll see a lot of terms such as GHG, Pools, Fluxes-- that probaby don't make sense to people from the technical background even after reading the Flint-wiki (Gets intimidating).
> 
> So you could maybe break it down for people like you and me? The Eli5 (Explain like I am 5) way.
> 
> It doesn't have to be all at once. Just pick up a few terms and try to tackle them.
> 
> (And later there could be follow up blogs with more terms.)
> 
> Cc: @shubhamkarande13 @aornugent @HarshCasper

Yeah sure.

**Comment by Sukriti-sood** [comment]
> Hi @Sukriti-sood
> 
> Here's a cool paper by the Werner Kurz, chair of the Technical Steering Committee, shared by @aornugent on the #outreachy channel of Slack:
> 
> [annurev-environ-012320-054809.pdf](https://github.com/moja-global/community-website/files/7728817/annurev-environ-012320-054809.pdf)

Thanks for Sharing, Should I work on this issue or its not needed ?

**Comment by shubhamkarande13** [comment]
@HarshCasper you're right. Maybe @aornugent could help us with some more blog ideas.
